vim/autoload/himalaya/msg.vim

```diff
@@ -375,7 +376,7 @@ function! s:bufwidth()
 endfunction
 
 function! s:get_msg_id(line)
-    return matchstr(a:line, '[0-9]*')
+  return matchstr(a:line, '[0-9a-z]*')
```

Comment:
In fact it should be `[0-9a-fA-F]*`, see https://datatracker.ietf.org/doc/html/rfc4122#section-3.